packages/react-mutation-mapper/src/component/civic/Civic.tsx

```diff
@@ -19,46 +25,46 @@ export function hideArrow(tooltipEl: any) {
     arrowEl.style.display = 'none';
 }
 
-@observer
-export default class Civic extends React.Component<ICivicProps, {}>
+export function sortValue(civicEntry:ICivicEntry | null | undefined): number
 {
-    @observable tooltipDataLoadComplete:boolean = false;
-    
-    public static sortValue(civicEntry:ICivicEntry | null | undefined): number
-    {
-        let score: number = 0;
+    let score = 0;
 
-        if (civicEntry) {
-            score = 1;
-        }
-
-        return score;
+    if (civicEntry) {
+        score = 1;
     }
 
-    public static download(civicEntry:ICivicEntry | null | undefined): string
-    {
-        if (!civicEntry) {
-            return "NA";
-        }
+    return score;
+}
 
-        const variants = _.values(civicEntry.variants);
-        const values: string[] = [];
+export function download(civicEntry:ICivicEntry | null | undefined): string
+{
+    if (!civicEntry) {
+        return "NA";
+    }
 
-        if (variants && variants.length > 0 && variants[0].evidence)
-        {
-            _.toPairs(variants[0].evidence).forEach(pair => {
-                values.push(`${pair[0]}: ${pair[1]}`);
-            });
-        }
+    const variants = _.values(civicEntry.variants);
+    const values: string[] = [];
 
-        // TODO actually this indicates that we have an entry but the evidence is empty
-        if (values.length === 0) {
-            return "NA";
-        }
+    if (variants && variants.length > 0 && variants[0].evidence)
+    {
+        _.toPairs(variants[0].evidence).forEach(pair => {
```

Comment:
why bother with toPairs?  _.forEach() can iterate over object's members, passing key and value to callback

Reply:
I haven't actually refactored the old implementation, just made the column compatible with `react-mutation-mapper`. Although this line seems like it is newly added, it is probably due to some cleanup / indentation change. Here is the original code: https://github.com/cBioPortal/cbioportal-frontend/pull/2981/files#diff-29ff9c92a56351a845f83f8ab38f6cd0L49